controllers/gcpcluster_controller.go

```diff
@@ -231,8 +231,8 @@ func (r *GCPClusterReconciler) reconcileDelete(ctx context.Context, clusterScope
 	log.Info("Reconciling Delete GCPCluster")
 
 	reconcilers := []cloud.Reconciler{
-		subnets.New(clusterScope),
 		loadbalancers.New(clusterScope),
+		subnets.New(clusterScope),
 		firewalls.New(clusterScope),
```

Comment:
should we delete the firewalls before as well?

Reply:
I don't think that's necessary as there are no conflicting resources being used by the firewalls.